src/pages/ItemDetail.js

```diff
@@ -0,0 +1,24 @@
+import React,{useContext} from 'react';
+import {Context} from '../context/Context';
+import Ratings from '../components/Ratings';
+import Price from '../components/Price';
+
+export default function ItemDetail() {
+    const {itemDetail} = useContext(Context);
+
+    console.log("itemdeet",itemDetail);
+    return (
+        <div>
+            {itemDetail.map(item=>{
```

Comment:
This wouldn't need to be a map if you don't store the item detail as an array of one item